src/Components/Header.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { Nav, NavItem } from "reactstrap"
 import { NavLink } from "react-router-dom"
-import dogLogo from "../assets/dogLogo.jpg"
+import dogLogo from "../Assets/dogLogo.jpg"
```

Comment:
since we are in React, let's allow pascal case to represent our classes/components. Since assets is a folder, it should be lowercase. Please modify it throughout your application.